packages/pluggableWidgets/accordion-web/src/Accordion.tsx

```diff
@@ -1,12 +1,12 @@
 import { createElement, ReactElement, useMemo } from "react";
 
-import AccordionComponent from "./components/Accordion";
+import AccordionComponent, { AccordionGroups } from "./components/Accordion";
```

Comment:
Seems a bit weird the imports as you have more than one export.. also commented in Accordion.tsx

Reply:
Changed it.